### PR TITLE
Fixes #37476 - update ak results in hostgroup

### PR DESCRIPTION
**Pull request by MariaAga**
selecting lifecycle env does not update the ak results in new host group, since useMemo doesnt update.
placeholder prop was wrong and should be placeholderText to show empty results

### CODE REVIEW

**Comment by adamruzicka** [comment]
Can't really speak to what goes on in the code, but it seems to resolve the issue

File: webpack/components/ActivationKeysSearch/index.js [hunk 1]
```diff
@@ -23,24 +23,26 @@ import { selectAPIStatus } from 'foremanReact/redux/API/APISelectors';
 const ActivationKeysSearch = () => {
   const ACTIVATION_KEYS = 'ACTIVATION_KEYS';
   const KT_AK_LABEL = 'kt_activation_keys';
-  const selectedEnvId = useMemo(() => {
+  const getSelectedEnvId = () => {
```
Comment (jeremylenz) [inline webpack/components/ActivationKeysSearch/index.js:26]: This seems to be vanilla JS, can we move it outside the component so the function isn't recreated on each render? That may provide the same benefit that the useMemo was supposed to.

Comment (MariaAga) [reply webpack/components/ActivationKeysSearch/index.js]: yeah, thats a better place for it

File: webpack/components/ActivationKeysSearch/index.js [hunk 2]
```diff
@@ -20,27 +20,29 @@ import { foremanUrl } from 'foremanReact/common/helpers';
...
-    const selectElement = document.querySelector('#hostgroup_content_view_id');
-    const selectedOption = selectElement.options[selectElement.selectedIndex];
-    let dataId = selectedOption?.getAttribute?.('data-id');
-    if (!dataId) {
-      dataId = selectElement.value;
-    }
-    return dataId;
-  }, []);
+  const [selectedEnvId, setSelectedEnvId] = useState(getSelectedEnvId());
+  const [selectedContentViewId, setSelectedContentViewId] = useState(getSelectedContentViewId);
```
Comment (jeremylenz) [inline webpack/components/ActivationKeysSearch/index.js:45]:
```suggestion
  const [selectedContentViewId, setSelectedContentViewId] = useState(getSelectedContentViewId());
```

**Review by jeremylenz — COMMENTED** [review]
Works well, just one more comment below

File: webpack/components/ActivationKeysSearch/index.js (hunk 2 above)
Comment (MariaAga) [reply webpack/components/ActivationKeysSearch/index.js]: Fixed, thanks

**Review by jeremylenz — APPROVED** [review]
Thanks @MariaAga! ACK

**Pull request merged**